Source/WebCore/style/StyleInvalidator.cpp

```diff
@@ -421,7 +421,7 @@ void Invalidator::invalidateShadowPseudoElements(ShadowRoot& shadowRoot)
         return;
 
     for (auto& descendant : descendantsOfType<Element>(shadowRoot)) {
-        auto& partId = descendant.userAgentPartId();
+        auto& partId = descendant.pseudo();
         if (!partId)
```

Comment:
You could rename these variables to `pseudo` or something.

Reply:
I think we might want to rename pseudo to userAgentPart at some point. I suppose I could try to tackle that here...

Reply:
I think I prefer leaving this to a removal of the wrapper and then we can rename all the things in a follow-up.

Reply:
Yeah that would make sense.